src/components/dropdowns/dropdown/DropdownCSS.js

```diff
@@ -0,0 +1,34 @@
+import { colors } from "../../colors/index";
+
+export const dropdownWrapperCSS = `
+	font-family: "HK Grotesk", sans-serif;
+	border-radius: 5px;
+	border-style: solid;
+	border-width: 1px;
+	border-color: ${colors.emerald400};
+`
+
+export const dropdownOuterCSS = `
+	padding: 12px;
+	font-weight: 700;
+	font-size: 18px;
+	background: ${colors.white};
+	color: black;
+	display: flex;
+	justify-content: space-between;
+	border-radius: 5px 5px 0 0;
+`
+
+export const dropdownInnerCSS = `
+	font-size: 14px;
+	font-weight: 400;
+	padding: 12px;
+	color: ${colors.white};
+	background: ${colors.midnight400};
```

Comment:
looks like the color's slightly off? would add `#4B406D` to colors.js and use it here